tests/test_all_rules.py

```diff
@@ -268,3 +269,27 @@ def test_required_tags(self):
 
             if missing_required_tags or is_missing_any_tags:
                 self.fail(error_msg)
+
+
+class TestRuleFiles(unittest.TestCase):
+    """Test the expected file names."""
+
+    def test_rule_file_names_by_tactic(self):
+        """Test to ensure rule files have the primary tactic prepended to the filename."""
+        rules = rule_loader.load_rules().values()
+
+        for rule in rules:
+            rule_path = Path(rule.path).resolve()
+            filename = rule_path.name
+
+            if rule_path.parent.name == 'ml':
+                continue
+
+            threat = rule.contents.get('threat')
+
+            if threat:
```

Comment:
```suggestion
            threat = rule.contents.get('threat')
            author = rule.contents.get('author')

            if threat and author and 'Elastic' in author:
```

Work for you @rw-access?

Reply:
good catch. forgot about the array.
 minor tweak:
```suggestion
            threat = rule.contents.get('threat', [])
            authors = rule.contents.get('author', [])
            
            if threat and 'Elastic' in authors:
```

Reply:
wfm 👍